File: platforms/iPad/Accounter iOS 1.0.2/Accounter/Record.m
//
//  Record.m
//  Accounter iOS2
//
//  Created by Amrit Mishra on 11/11/11.
//  Copyright 2011 __MyCompanyName__. All rights reserved.
//

#import "Record.h"


@implementation Record

-(id)init{
    
    self = [super init];
    if (self) {
        cells = [[NSMutableArray alloc]init];

    }
    return self;
}

- (NSString *)getCode{
    return code;
}

- (NSMutableArray *)getCells{
    return cells;
}

- (void)setCells:(NSMutableArray *)cellNames{
    cells = [[NSMutableArray alloc]initWithArray:cellNames];
}

- (void)setCode:(NSString *)stringCode{
    code = stringCode;
}

@end

File: platforms/blackberry/Accounter/src/com/vimukti/accounterbb/result/Record.java
package com.vimukti.accounterbb.result;

import java.util.Vector;

public class Record {
	private String code;
	private Vector cells = new Vector();

	public void setCells(Vector cells) {
		this.cells = cells;
	}

	public Vector getCells() {
		return cells;
	}

	public void setCode(String code) {
		this.code = code;
	}

	public String getCode() {
		return code;
	}
}
